genesis/process/disabled/disabledTxLogProcessor.go

```diff
@@ -0,0 +1,27 @@
+package disabled
+
+import (
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/data"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+)
+
+// DisabledTxLogProcessor represents a disabled transaction log processor that does nothing
+type DisabledTxLogProcessor struct {
```

Comment:
i would prefer with normal log processor - as in case of delegation is is a must :)

Reply:
That is how it was before, it used a mock. can be checked on development branch

Reply:
injected the same tx logs processor

Reply:
I know it was the same :).